user_input.pl

```diff
@@ -74,10 +74,10 @@
     repeat,
     (
         read(UserInput),
-        UpperLimit is 65 + BoardSize,
-        between(65, UpperLimit, UserInput),
-        letter_to_col_num(UserInput, Col), !;
-        writeln("Invalid input. Please enter a valid column letter."),
+        atom(UserInput),
+        letter_to_col_num(UserInput, Col),
+        between(1, BoardSize, Col), !;
+        writeln("Invalid input. Please enter a valid column letter (inside brackets, like 'A')."),
```

Comment:
Not so sure what you did here.. The user is now expected to type 'A'? Anyway, I started to fix this in my branch.
Also, the message is unclear because brackets are (), not ''